src/_layouts/default.html

```diff
@@ -7,13 +7,12 @@
 
     {%- include header.html -%}
 
-    <main class="page-content" aria-label="Content">
-      <div class="wrapper">
-        {{ content }}
-      </div>
+    <main class="page-content" aria-label="Content" id="content">
+      {{ content }}
     </main>
 
-    <!-- {%- include footer.html -%} -->
+    {%- include footer.html -%}
```

Comment:
Another tangential comment/question: should we include the footer on the "home" landing page as well?

The easiest way I've found to do this is to add `{%- include footer.html -%}` at the end of `index.md`, but it seems a bit inelegant...

Reply:
I had intended to make the landing page inherit from the same default page template as all the other pages (for example, the header bar is a different height on the landing page at present), which would include header, navigation and footer.

Reply:
I think it's okay to have a separate template and a slightly different design for the landing page. It depends on if we're thinking of that as a home page or a landing page - not quite the same thing. Either way, I think we should have the footer on the home/landing page.